Documentation/tutorials/edge-bridge-tutorial.md

```diff
@@ -0,0 +1,269 @@
+    - If using a physical device, it may be helpful to have a way to send this link to the device (ex: Airdrop, email, text, etc.)
+2. Open the sample app and tap the Assurance button
+3. Paste the Assurance session link copied from step 1 and tap `Connect`
+    - If using the simulator, it is possible to enable the paste menu by clicking in the text field twice, with a slight pause between clicks.
+4. App should open and show the Assurance PIN screen to authenticate the session connection; enter the PIN from the session details and tap `Connect`
+
+Once connected to Assurance, an Adobe Experience Platform icon will appear in the top right corner of the screen with a green dot indicating a connected session. In the web-based Assurance session, there is also an indicator in the top right that shows the number of connected sessions (which in this case should now show a green dot with "1 Client Connected", marked `1` in the screenshot below).
+
+<img src="../assets/edge-bridge-tutorial/simulator-assurance-connection.jpg" alt="Assurance Session Start - iOS simulator" width="400"/>
+<img src="../assets/edge-bridge-tutorial/assurance-session-start.jpg" alt="Assurance Session Start - Web UI after connection" width="800"/>  
+
+Observe how in the Assurance session Events view (`2`), there are already events populating as a consequence of the connection of the mobile app to the Assurance session (`3`); the Assurance extension itself emits events about the session connection and subsequently captures these events to display in the web-based session viewer. You can expect Assurance to capture all events processed by the AEP SDK from all other extensions as well.  
+
+### 3. Event transactions view - check for Edge Bridge events  
+In order to see Edge Bridge events, in the connected app instance, trigger a `trackAction` and/or `trackState` within the app which the Edge Bridge extension will convert into Edge events. This event will be captured by the Assurance extension and shown in the web session viewer.
+
+```swift
+Button("Track Action", action: {
+    // Dispatch an Analytics track action event which is handled by the
+    // Edge Bridge extension which forwards it to the Edge Network.
+
+    let data: [String: Any] = [
+        "product.id": "12345", 
+        "product.add.event": "1", 
+        "product.name": "wide_brim_sunhat", 
+        "product.units": "1"
+    ]
+    MobileCore.track(action: "add_to_cart", data: data)
+}).padding()
+```
+
+```swift
+Button("Track State", action: {
+    // Dispatch an Analytics track state event which is handled by the
+    // Edge Bridge extension which forwards it to the Edge Network.
+
+    let data: [String: Any] = [
+        "product.name": "wide_brim_sunhat", 
+        "product.id": "12345", 
+        "product.view.event": "1"
+    ]
+    MobileCore.track(state: "hats/sunhat/wide_brim_sunhat_id12345", data: data)
+}).padding()
+```
```

Comment:
As these are already implemented in the app, and that this section is about using the Assurance UI, the code samples don't need to be included here. Having screen shots showing this data in Assurance is sufficient.